src/syscall_mocking/mod.rs

```diff
@@ -40,6 +42,34 @@ impl Tracer {
         }
     }
 
+    fn execve(
```

Comment:
because nicer error messages increase complexity of the closure given to `fork_with_child_errors`, this has been outlined.